### DISCUSSION TITLE
8.2.0 Release - The Renamening

### DISCUSSION THREAD

**Discussion by Donnnno** [Arcticons-Team/Arcticons#1836] [Announcements]
## 8.2.0 - The Renamening

* 🔠 Sometimes you have a good idea, and sometimes it just doesn't work that good in practice. That was the case with the Arcticons names. Dark and Light stands for Darkmode and Lightmode, but that was confusing, because it wasn't in the title. We've renamed it to Arcticons, Arcticons Black, and Arcticons Material You.
* 🎉 150 new and updated icons! (thanks Orbitaldr0p, Supertsy5, MikeTheGreat, TotallyAvailable, Xprateek, Subroutine7901, Ojppe, Lavin-Tom, Quantenzitrone)
* ✨ Updates to make sure lots of apps work again.
* 🔥 8200 icons in total!

![Untitled-1](https://github.com/Donnnno/Arcticons/assets/31142286/83eacfe5-f49d-4ae5-82d0-61e56658a117)
![Untitled-12](https://github.com/Donnnno/Arcticons/assets/31142286/ced26f89-71ee-4396-9251-a0fefcdb08d1)

## Auto notes:
* Update & extend appfilter.xml by @TotallyAvailable in https://github.com/Donnnno/Arcticons/pull/1785
* The renamening by @Donnnno in https://github.com/Donnnno/Arcticons/pull/1788
* Update & Extend Coverage by @TotallyAvailable in https://github.com/Donnnno/Arcticons/pull/1786
* Fix typo by @supertsy5 in https://github.com/Donnnno/Arcticons/pull/1787
* Update appfilter.xml by @onuronsekiz in https://github.com/Donnnno/Arcticons/pull/1782
* Couple more fixed & covered by @TotallyAvailable in https://github.com/Donnnno/Arcticons/pull/1789
* Fix Missing and extended Coverage by @TotallyAvailable in https://github.com/Donnnno/Arcticons/pull/1791
* Icons by @Kaiserdragon2 in https://github.com/Donnnno/Arcticons/pull/1784
* Update appfilter.xml by @Lavin-tom in https://github.com/Donnnno/Arcticons/pull/1802
* update one appfilter entry by @dgcampbe in https://github.com/Donnnno/Arcticons/pull/1806
* 12 New Icons by @onuronsekiz in https://github.com/Donnnno/Arcticons/pull/1792
* Small Update  by @TotallyAvailable in https://github.com/Donnnno/Arcticons/pull/1807
* Added icon for China Mobile by @supertsy5 in https://github.com/Donnnno/Arcticons/pull/1793
* 8 icons, a few fixes by @dioxias in https://github.com/Donnnno/Arcticons/pull/1796
* Added icon for FAMILIES Talking Points by @MikeTheGreat in https://github.com/Donnnno/Arcticons/pull/1798
* fix #1794: errors in dropbox print and signal icons by @quantenzitrone in https://github.com/Donnnno/Arcticons/pull/1795
* Added missing entries for discord's new app icons by @Neop0litan in https://github.com/Donnnno/Arcticons/pull/1797
* Update appfilter by @ojppe in https://github.com/Donnnno/Arcticons/pull/1810
* a better mull icon by @Donnnno in https://github.com/Donnnno/Arcticons/pull/1809
* New Icons by @onuronsekiz in https://github.com/Donnnno/Arcticons/pull/1811
* 26 New Icons by @onuronsekiz in https://github.com/Donnnno/Arcticons/pull/1815
* Added more icons by @MikeTheGreat in https://github.com/Donnnno/Arcticons/pull/1817
* Remove Google Weather Shortcut  by @Kaiserdragon2 in https://github.com/Donnnno/Arcticons/pull/1820
* Fix ZipXtract by @TotallyAvailable in https://github.com/Donnnno/Arcticons/pull/1823
* re-cover Alga by @TotallyAvailable in https://github.com/Donnnno/Arcticons/pull/1827
* Added COC2 & Cytoid Icons by @orbitaldr0p in https://github.com/Donnnno/Arcticons/pull/1830
* Add Soundbound Icon by @Subroutine7901 in https://github.com/Donnnno/Arcticons/pull/1819
* Added AirQualityWa icon (resubmit) by @MikeTheGreat in https://github.com/Donnnno/Arcticons/pull/1829
* Added some icons by @xprateek in https://github.com/Donnnno/Arcticons/pull/1826
* New icons by @Donnnno in https://github.com/Donnnno/Arcticons/pull/1818

## New Contributors
* @TotallyAvailable made their first contribution in https://github.com/Donnnno/Arcticons/pull/1785
* @supertsy5 made their first contribution in https://github.com/Donnnno/Arcticons/pull/1787
* @Neop0litan made their first contribution in https://github.com/Donnnno/Arcticons/pull/1797
* @Subroutine7901 made their first contribution in https://github.com/Donnnno/Arcticons/pull/1819

**Full Changelog**: https://github.com/Donnnno/Arcticons/compare/8.0.6...8.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Donnnno/Arcticons/releases/tag/8.2.0'>8.2.0 Release - The Renamening</a>.</em>

**Comment by xprateek** [comment]
![Screenshot_20231125-213501.png](https://github.com/Donnnno/Arcticons/assets/47496067/b0f0675e-89e6-4f49-8b15-0dea612ab6df)

Antiiq Icon! Drawables not generated

**Reply by Donnnno** [reply]
In the dark version? It shows the icon for me :)

**Reply by xprateek** [reply]
In Material You!

**Reply by Donnnno** [reply]
fixed it, check the new downloads :)

**Comment by quantenzitrone** [comment]
> We've renamed it to Arcticons, Arcticons Black, and Arcticons Material You.

![image](https://github.com/Donnnno/Arcticons/assets/74491719/ad449a18-abde-4227-b1b8-a3518eae08cd)

**Reply by Donnnno** [reply]
😅

**Reply by Donnnno** [reply]
![image](https://github.com/Donnnno/Arcticons/assets/31142286/f4e4cbac-4cf1-4254-8126-de93a9f983f6)

**Comment by Donnnno** [comment]
Typo in Google Chrome: Goole Chrome

oops

**Reply by Donnnno** [reply]
fixed

**Comment by onuronsekiz** [comment]
My heart is broken, I was not thanked :'(

**Reply by Donnnno** [reply]
Oh nooooo! Shit :( 

fixed it in on the releases page

**Reply by onuronsekiz** [reply]
dude I was joking :))

**Reply by Donnnno** [reply]
Gotta credit everyone properly for all the hard work :)

**Reply by xprateek** [reply]
![Screenshot_20231127-120215.png](https://github.com/Donnnno/Arcticons/assets/47496067/21f41b84-8a80-4d96-8bce-9344e231eaae)

Tata_neu icon in Material You! @Donnnno

**Reply by Kaiserdragon2** [reply]
> ![Screenshot_20231127-120215.png](https://github.com/Donnnno/Arcticons/assets/47496067/21f41b84-8a80-4d96-8bce-9344e231eaae)
> 
> Tata_neu icon in Material You! @Donnnno 
> 
> 
Things like this should be fixed with 
#1838
#1824
